config/locales/en/shared.yml

```diff
@@ -37,6 +37,24 @@ en:
             check_box_child_care: Childcare payments
             check_box_maintenance_out: Maintenance payments to a former partner
             check_box_legal_aid: Payments towards legal aid in a criminal case
+        providers:
+          cash_incomes:
+            hint: Enter the total cash amount you received in each month, or enter 0 if you received none.
```

Comment:
as these are under providers, should they refer to the client i.e rather that 'you'